qboost.py

```diff
@@ -284,6 +284,28 @@ def __init__(self, X, y, lam, weak_clf_scale=None, drop_unused=True):
         super().__init__(weak_classifiers, weights, weak_clf_scale)
         self.fit_offset(X)
 
+        # Save candidates so we can provide a baseline accuracy report.
+        self._wclf_candidates = wclf_candidates
+
+    def report_baseline(self, X, y):
+        """Report accuracy of weak classifiers.
+
+        This provides context for interpreting the performance of the boosted
+        classifier.
+        """
+
+        from tabulate import tabulate
```

Comment:
It would probably be better to put the import at the top of the file for consistency.

The other thing is that it looks like the `tabulate` package is not currently installed in our docker image for Leap IDE, which means the demo wouldn't run there (at least with the `--verbose` option).

Reply:
Ah, the IDE.  I did not think of that.  I put the import in the function so it would not be required unless the `--verbose` option is used.  I'll just remove the dependency, it's easily replaced.  Thanks!

Reply:
We can easily add this (and other) deps to our IDE docker image, especially if they are broadly useful.

For specific, one-off deps, we should leverage [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file/), where they can be installed during [the init step](https://www.gitpod.io/docs/config-start-tasks/) (for an example, see [here](https://github.com/dwave-examples/image-segmentation/), where we install opencv). Since we have [workspace prebuilds](https://www.gitpod.io/docs/prebuilds/) turned on in LeapIDE prod, there's no noticeable end-user impact on load/init time.

Reply:
Done.

Reply:
Cool, I didn't know about that.